Projects/UOContent/Engines/ConPVP/DuelContext.cs

```diff
@@ -2070,13 +2070,17 @@ public void ClearIllegalItems(Mobile mob)
                     pack.DropItem(item);
 
                     // TODO: Use Layer instead of Type
-                    int number = item switch
+                    var number = item switch
```

Comment:
Per conversation, maybe this?
```cs
                    int number = item switch
                    {
                        BaseWeapon _ => 1062001, // You can no longer wield your ~1_WEAPON~
                        _ when !(item is BaseShield) && (item is BaseArmor || item is BaseClothing) => 1062002 // You can no longer wear your ~1_ARMOR~
                        _ => 1062003, // You can no longer equip your ~1_SHIELD~
                    };
```

Reply:
That looks better, and not at the same time. But we should go with that. Adjusted the code.